src/stories/components/Form/styles.scss

```diff
@@ -0,0 +1,20 @@
+@import '../../../scss/variables.scss';
+
+// @file
+// Form
+
+.form__container{
+    display: flex;
```

Comment:
Any reason why you choose to use `display: flex`? If it's just because of the `Submit` button, I can show you how to align it right without having to call flexbox.

Reply:
You are right, I didn't need to use display flex for the submit button, your solution fixed it